x/exchange/keeper/proposal_handler_test.go

```diff
@@ -35,7 +36,8 @@ func (s *KeeperTestSuite) TestPoolParameterChangeProposal() {
 	proposal = types.NewMarketParameterChangeProposal(
 		"Title", "Description", []types.MarketParameterChange{
 			types.NewMarketParameterChange(
-				3, utils.ParseDec("0.001"), utils.ParseDec("0.002"), utils.ParseDec("0.5")),
+				3, utils.ParseDec("0.001"), utils.ParseDec("0.002"), utils.ParseDec("0.5"),
+				nil, nil, nil, nil),
 		})
 	s.Require().NoError(proposal.ValidateBasic())
 	s.Require().EqualError(handler(s.Ctx, proposal), "market 3 not found: not found")
```

Comment:
How about having one more test as below?
```
func (s *KeeperTestSuite) TestPoolParameterChangeProposal_WrongFees() {
 ...
}
```

Reply:
Wrong fees will be rejected before reaching the proposal handler, by `ValidateBasic`.